### PR TITLE
Release of version 11.1.0

### PR DESCRIPTION
**Pull request by grossmannmartin**
| Q             | A
| ------------- | ---
|Description, reason for the PR| New version is here 🎉 
|New feature| Yes 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| Yes 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,60 @@ Packages are formatted by release version. You can see all the changes done to p
...
+* [shopsys] added ability to change periodicity and length of cron runs by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2584
+* [framework] columns in grid can now be reordered by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2564
+* [shopsys] added ability to change content of robots.txt file through administration by @radek-bruha in https://github.com/shopsys/shopsys/pull/2591
+* [shopsys] users and admins now have only five tries to log in one minute by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2599
+* [framework] cron overview is now available for admin in the default state by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2600
+* [shopsys] added ability to change email whitelist through administration by @radek-bruha in https://github.com/shopsys/shopsys/pull/2592
+* [shopsys] don't show 500 error page when you have 405 http code by @sspooky13 in https://github.com/shopsys/shopsys/pull/2666
+
+### :bug: Bug Fixes
+* [project-base] returned symfony.lock to project-base as it caused problems with symfony flex installing packages again by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2595
```
Comment (TomasLudvik) [inline CHANGELOG.md:43]: I would not mention this as it matters only for releaser

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,60 @@ Packages are formatted by release version. You can see all the changes done to p
...
+* [framework] AdministrationFacade: support requests without _route parameter by @mposchl in https://github.com/shopsys/shopsys/pull/2373
+* [shopsys] updated acceptance tests because they were no longer working locally by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2610
+* [shopsys] too many attempts js error is now translated on frontend by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2613
+* [framework] removed class duplication in expanded choice by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2614
+
+### :hammer: Developer experience and refactoring
+* [shopsys] removed duplicated CreateReleaseNotesReleaseWorker by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2579
```
Comment (TomasLudvik) [inline CHANGELOG.md:50]: Again releaser

File: CHANGELOG.md [hunk 3]
```diff
@@ -26,6 +26,60 @@ Packages are formatted by release version. You can see all the changes done to p
...
+* [framework] added missing getters to UploadedFile by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2582
+* [framework] bin/console base phing targets are now more verbose by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/2507
+* [framework] Transport::getPrice() - fix error message by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/2506
+* [framework] added commonly used methods to Price class by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2589
+* [framework] build-demo-dev-quick phing target now correctly recreates elasticsearch structure by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2590
+* [framework] removed DRY violation in AggregationResultToProductFilterCountDtaTransformer and fixed property name by @sspooky13 in https://github.com/shopsys/shopsys/pull/2501
+* [shopsys] checking if order status update email should be sent is done directly in OrderMailFacade instead in multiple places by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2588
+* [framework] fixed upgrade notes and resolved possible BC-break by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2598
```
Comment (TomasLudvik) [inline CHANGELOG.md:58]: I would not mention this

File: CHANGELOG.md [hunk 4]
```diff
@@ -26,6 +26,60 @@ Packages are formatted by release version. You can see all the changes done to p
...
+* [shopsys] updated installation docs for macOS using Mutagen by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2593
+* [shopsys] added BlankLineBeforeStatementFixer for ensuring empty line before statements by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2128
+* [framework] RegisterProjectFrameworkClassExtensionCompilerPass now ensures that both class and interface alias are registered in Symfony services by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2605
```
Comment (TomasLudvik) [inline CHANGELOG.md:61]: Do we want to add that fix, that only app interfaces are replaced before release?

File: CHANGELOG.md [hunk 5]
```diff
@@ -26,6 +26,60 @@ Packages are formatted by release version. You can see all the changes done to p
...
+* [project-base] config folder is now less confusing by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2607
+
+### :link: Frontend API
+* authentication failure now return errors as array by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2387
+* empty strings are now considered nulls in frontend API by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2601
```
Comment (TomasLudvik) [inline CHANGELOG.md:66]: These two are missing bracketed package that it belongs to - do we want to add it or remove it everywhere?

File: upgrade/UPGRADE-v11.1.0.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
-# [Upgrade from v11.0.0 to v11.0.1-dev](https://github.com/shopsys/shopsys/compare/v11.0.0...master)
+# [Upgrade from v11.0.0 to v11.1.0](https://github.com/shopsys/shopsys/compare/v11.0.0...v11.1.0)
 
-This guide contains instructions to upgrade from version v11.0.0 to v11.0.1-dev.
+This guide contains instructions to upgrade from version v11.0.0 to v11.1.0.
 
-**Before you start, don't forget to take a look at [general instructions](https://github.com/shopsys/shopsys/blob/7.3/UPGRADE.md) about upgrading.**
+**Before you start, don't forget to take a look at [general instructions](https://github.com/shopsys/shopsys/blob/v11.1.0/UPGRADE.md) about upgrading.**
```
Comment (TomasLudvik) [inline upgrade/UPGRADE-v11.1.0.md:5]: I think that v should not be in the link - can you fix the releaser step please?

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: upgrade/UPGRADE-v11.1.0.md (hunk 6 above)
Comment (grossmannmartin) [reply upgrade/UPGRADE-v11.1.0.md:5]: I have done it manually... the released tag is with `v`, so the link will be all right after the release... Another option is to have the link to the branch, but that would be `master` in this case, and master may be something different in the near future 🤷

File: CHANGELOG.md (hunk 5 above)
Comment (grossmannmartin) [reply CHANGELOG.md]: added in c110a6e27

File: CHANGELOG.md (hunk 2 above)
Comment (grossmannmartin) [reply CHANGELOG.md]: removed in c110a6e27

File: CHANGELOG.md (hunk 1 above)
Comment (grossmannmartin) [reply CHANGELOG.md]: removed in c110a6e27

File: CHANGELOG.md (hunk 4 above)
Comment (grossmannmartin) [reply CHANGELOG.md:58]: added in #2681

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**